.travis.yml

```diff
@@ -27,10 +27,14 @@ install:
 
 env:
   - NXF_VER='0.32.0' # Specify a minimum NF version that should be tested and work
-  - NXF_VER='' # Plus: get the latest NF version and check that it works
+  # - NXF_VER='' # Plus: get the latest NF version and check that it works
```

Comment:
I know that tests are longer with that, but, it's quite useful to know if it works for newer versions as well

Reply:
ok.